frontend/src/globals.d.ts

```diff
@@ -0,0 +1,2 @@
+declare module "react-slippy-map";
+declare module "google-map-react";
```

Comment:
You don't need this file at all (delete it) because:
1. You're not using `react-slippy-map` any more
2. `google-map-react` has type declarations ([the npm page](https://www.npmjs.com/package/google-map-react) has a "DT" next to the name), so you should run `npm install --save-dev @types/google-map-react`

Reply:
Deleted slippy map declaration, but the code breaks if we take the other one out

![unknown](https://user-images.githubusercontent.com/110834537/197781064-5bcfe4ea-4b53-4a08-9c07-9ab954f82d40.png)